libr/anal/fcn.c

```diff
@@ -880,8 +880,8 @@ static int fcn_recurse(RAnal *anal, RAnalFunction *fcn, ut64 addr, ut8 *buf, ut6
 						ls_free (secs);
 					}
 					if (s) {
-						must_eob = (s->vaddr <= op.jump);
-						must_eob &= ((s->vaddr + s->vsize) > op.jump);
+						must_eob = (s->vaddr > op.jump);
+						must_eob |= ((s->vaddr + s->vsize) < op.jump);
```

Comment:
`<=`

There is a regression introduced at https://github.com/radare/radare2/commit/0161fa0ec2d1f20f38873d6a48ae13e08e2cc138#diff-49a52329a1b3f5e5f62d571577391074L880